rules/subrepo_rules.build_defs

```diff
@@ -173,3 +173,13 @@ def github_repo(name:str, repo:str, revision:str, build_file:str=None, hashes:st
         config = config,
         bazel_compat = bazel_compat,
     )
+
+def arch(name:str, os:str, arch:str):
```

Comment:
can you add a docstring please? that gets parsed to generate output for `plz help` & the docs etc.